presto-main/src/main/java/io/prestosql/sql/rewrite/ShowStatsRewrite.java

```diff
@@ -414,4 +225,12 @@ private static Expression toStringLiteral(Type type, double value)
             throw new IllegalArgumentException("Unexpected type: " + type);
         }
     }
+
+    private static Expression toDoubleLiteral(double value)
+    {
+        if (!isFinite(value)) {
+            return NULL_DOUBLE;
```

Comment:
This should be more contextual. 
`null` makes sense 
- for min being -inf
- for max being +inf
- for row count, NDV, null fraction, data size being nan

but we probably should not output dull null eg when
- row count is +inf

please maybe add a todo note?


same in `toStringLiteral(Type type, double value)` above

Reply:
> I don't see a big difference between NaN and null from user perspective.

I do not see either. There can be a difference between +inf and null.